source/geod24/LocalRest.d

```diff
@@ -463,7 +463,7 @@ private bool isMainThread () @trusted nothrow
 
 public void runTask (void delegate() dg) nothrow
 {
-    assert(!isMainThread(), "Cannot call this function from the main thread");
+    assert(scheduler !is null, "Cannot call this delegate from the main thread");
```

Comment:
And `sleep` doesn't need this same change right?

Reply:
No, sleep should still require a running Fiber